apps/api/plane/celery.py

```diff
@@ -50,9 +50,21 @@
         "schedule": crontab(hour=2, minute=0),  # UTC 02:00
     },
     "check-every-day-to-delete-api-logs": {
-        "task": "plane.bgtasks.api_logs_task.delete_api_logs",
+        "task": "plane.bgtasks.cleanup_task.delete_api_logs",
         "schedule": crontab(hour=2, minute=30),  # UTC 02:30
     },
+    "check-every-day-to-delete-logs": {
```

Comment:
@pablohashescobar Let's change the name to include notification in it, to make it more specific